extensions/odoo_theme/static/scss/_variables.scss

```diff
@@ -149,68 +114,69 @@ $i-close: "\e942";
-$gray-darker:                               #282F33;
-
-$primary:                                   $color-alpha-dark;
-$secondary:                                 $o-violet-dark;
-
-$brand-primary :                            $o-violet;
-$brand-success :                            #50AF51;
-$brand-info:                                #4B9EB6;
-$brand-warning :                            #f39c1e;
-$brand-danger  :                            #D9534F;
-
-$doc_paper_dark:                            $gray-lighter;
-$doc_paper:                                 #ffffff;
-$doc_example:                               #519161;
-$doc_exercise:                              #938E94;
-$doc_code-bg:                               #F8F8F8;
-$doc_lime:                                  #CDDC39;
-$doc_orange:                                #FF5722;
-$doc_cyan:                                  $color-alpha;
+//Replace bootstrap's gray values
+$white:                                     #FFFFFF;
+$gray-100:                                  #F9F9F9 !default;
+$gray-200:                                  #F2F2F2 !default;
+$gray-300:                                  #DEE2E6 !default;
+$gray-400:                                  #D5D5D5 !default;
+$gray-500:                                  #ADB5BD !default;
+$gray-600:                                  #888888 !default;
+$gray-700:                                  #707070 !default;
+$gray-800:                                  #343A40 !default;
+$gray-900:                                  #282F33 !default;
+
+$o-light:                                   $white;
+$o-dark :                                   $gray-800;
+
+$o-gray-light :                             $gray-400;
+$o-gray :                                   $gray-600;
+
+$o-gray-border:                             $gray-400;
+$o-gray-bg:                                 $gray-100;
+$o-gray-color:                              $gray-900;
+
+$o-violet :                                 #7A436B; //Odoo's #875a7b Doesn't give AAA contrast on white background
+$o-blue :                                   #77b5d1;
+$o-cyan:                                    #0dcaf0;
+$o-teal :                                   #017e84;
+$o-green :                                  #519161;
+$o-orange:                                  #ffa930;
+$o-red:                                     #D9534F;
+$o-red-light:                               #FF5722;
+
+$blue:                                      $o-blue;
+$purple:                                    $o-violet;
+$cyan:                                      $o-cyan;
+$red:                                       $o-red;
+$orange:                                    $o-orange;
+$green:                                     $o-green;
+$teal:                                      $o-teal;
+
+$primary:                                   $o-teal;
+$secondary:                                 $o-violet;
+$light:                                     $o-light;
```

Comment:
this will generate  "white-bg buttons" classes, "white-bg" background classes etc.... not very useful I think

Why don't use `$gray-100`, the   `$o-gray-bg` **value** instead?
It will generate button and bg classes that we may actually use.. I see that  you're calling `$o-gray-bg` in several places